iwyu_preprocessor.cc

```diff
@@ -93,14 +93,11 @@ static string GetIncludeNameAsWritten(SourceLocation include_loc) {
   return GetIncludeNameAsWritten(include_loc, DefaultDataGetter());
 }
 
-// For a particular #include line that include_loc points to,
-// returns true if the given text occurs on the line.
-// (Case sensitive.)
-static bool IncludeLineHasText(SourceLocation include_loc,
-                               const string& text) {
-  const string data = GetSourceTextUntilEndOfLine(include_loc,
-                                                  DefaultDataGetter());
-  return data.find(text) != string::npos;
+bool LineHasText(SourceLocation source_location, StringRef text)
+{
+    const StringRef data = GetSourceTextUntilEndOfLine(source_location,
```

Comment:
clang-format makes hundreds of changes when run on these files, the vast bulk of which are not related to my changes. I'm using clang-format from my local build branched off of the Clang 4.00 tagged release with -style=file which should pick up the .clang-format in the include-what-you-use directory. I even tried moving it to a new directory so there was no parent .clang-format that could be picked up, only the one in include-what-you-use.
Am I using it wrong or has clang-format not been run on these for a long time?